src/components/VrxMenu/VrxMenu.mdx

```diff
@@ -0,0 +1,30 @@
+import {Meta} from "@storybook/blocks";
+
+<Meta title="VrxMenu/About" />
+
+# VireoXmenu
+
+VireoXmenu is a series of components that can be used to create a menu system for your web application.
+It is designed to be used with the component from VireoX library, but can also be used independently.
```

Comment:
The sentence isn't clear, is it designed to be used with any component from the library? Or a specific one (I.E. VrxLink)?

If with any, then the sentence would be plural:
- It is designed to be used with the `components`.....